crypto/pqt/pqt_kem.c

```diff
@@ -0,0 +1,512 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include "pqt_kem.h"
+
+#include <openssl/digest.h>
+#include <openssl/ecdh.h>
+#include <openssl/hkdf.h>
+#include <openssl/rand.h>
+#include "openssl/bn.h"
+
+// |nistp_internal_keygen_deterministic| needs this for montgomery reduction.
+#include "../fipsmodule/ec/internal.h"
+
+#include "../internal.h"
+
+// HKDF Labels
+#define PQT25519_LABEL "PQT255-v1"
+#define PQT256_LABEL "PQT256-v1"
+#define PQT384_LABEL "PQT384-v1"
+#define PQT_LABEL_LEN 9
+
+// 1. PQ KEM Wrappers
```

Comment:
Turns out I do not. I was using ML-KEM from `ml_kem.h` (which has the flipped return value) instead of `crypto/kem`. The latest version fixes this, and no longer has PQ wrappers.